_maps/RandomRuins/SpaceRuins/turretedoutpost.dmm

```diff
@@ -221,12 +218,6 @@
 /obj/machinery/door/airlock/public/glass,
 /turf/open/floor/plasteel/dark,
 /area/ruin/space/has_grav/turretedoutpost)
-"aJ" = (
-/turf/open/floor/plasteel{
-	dir = 4;
-	icon_state = "darkredfull"
```

Comment:
Used to have red turf decals on the floor here. Mind adding them?

Reply:
![image](https://user-images.githubusercontent.com/6209658/45307615-7223d580-b4ed-11e8-8551-d71557d46469.png)